ansible/roles/debops.phpipam/defaults/main.yml

```diff
@@ -10,240 +10,328 @@
 # .. include:: ../../../includes/global.rst
 
 
-# .. envvar:: phpipam_mode [[[
+# Basic configuration options [[[
+# -------------------------------
+
+# .. envvar:: phpipam__mode [[[
 #
 # What parts of the "phpIPAM stack" to enable:
 #
 # - ``webui``: configure the Web interface
 #
 # - ``scripts``: configure phpipam-scripts
 #
-phpipam_mode: [ 'webui', 'scripts' ]
+phpipam__mode: [ 'webui', 'scripts' ]
 
                                                                    # ]]]
-# ---- Basic configuration options ----
-
-# .. envvar:: phpipam_dependencies [[[
+# .. envvar:: phpipam__dependencies [[[
 #
 # Should phpipam role manage it's own dependencies?
-phpipam_dependencies: True
+phpipam__dependencies: True
+                                                                   # ]]]
 
+# .. envvar:: phpipam__php__dependent_packages [[[
+#
+# List of PHP packages to install using :ref:`debops.php` role.
+phpipam__php__dependent_packages:
+  - 'mysql'
+  - 'gmp'
+  - 'gd'
+  - 'curl'
+  - 'mcrypt'
+  - 'ldap'
+  - 'php-pear'
                                                                    # ]]]
-# .. envvar:: phpipam_domain [[[
+# .. envvar:: phpipam__domain [[[
 #
 # Domain on which phpIPAM is configured in nginx
-phpipam_domain: [ 'ipam.{{ ansible_domain }}' ]
+phpipam__domain: [ 'ipam.{{ ansible_domain }}' ]
```

Comment:
This should be a string, not a list. You should also rename this variable to `phpipam__fqdn` to better explain its purpose. Don't forget to update all of the references, and remote `[0]` as a list index from them.

Reply:
done